assets/package.json

```diff
@@ -67,7 +66,6 @@
     "jest": "^26.6.3",
     "mini-css-extract-plugin": "^2.6.0",
     "node-sass": "^7.0.1",
-    "optimize-css-assets-webpack-plugin": "^6.0.1",
```

Comment:
There's still a reference to this in the Webpack configuration, causing building the static content to fail.

Reply:
Also according to the `optimize-css-assets-webpack-plugin`, there's another plugin you can use with newer versions of Webpack called [css-minimizer-webpack-plugin](https://github.com/webpack-contrib/css-minimizer-webpack-plugin), so it might be worth switching to that rather than moving the plugin completely.

Followup question: was there some additional incompatibility with React Router 6, or was this because of Webpack 5? I'm surprised this didn't become an issue earlier (I upgraded from Webpack 4 to 5 just a few months ago).

Reply:
I installed css-minimizer-webpack-plugin and updated webpack.config.js in 07823ee8

Reply:
For what it's worth, you can try `npm run deploy` to try out the Webpack build without doing a full build of the Docker image.